index.html

```diff
@@ -1268,9 +1251,15 @@ <h2>Mapping for devices that do not support hover</h2>
             </div>
         </section>
     </section>
-    <section class='informative'>
+    <section>
         <h2>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitudeAngle</code> / <code>azimuthAngle</code></h2>
-        <p>Depending on the specific hardware and platform, user agents will likely only receive one set of values for the transducer orientation relative to the screen plane — either <code>tiltX</code> / <code>tiltY</code> or <code>altitudeAngle</code> / <code>azimuthAngle</code>. The following basic code provides an initial suggested approach for converting these values.</p>
+        <p>Pointer Events include two complementary sets of attributes to express the orientation of a transducer relative to the X-Y plane:
+            <code>tiltX</code> / <code>tiltY</code> (introduced in the original Pointer Events specification), and
+            <code>azimuthAngle</code> / <code>altitudeAngle</code>
+            (adopted from the <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification).</p>
```

Comment:
I think we need to retain the next line here ("Implementations SHOULD provide both sets of attributes for trusted events").  This would become normative of course!

Reply:
decided that this is implicit as it's part of the IDL in today's meeting